hyperspy/drawing/image.py

```diff
@@ -62,6 +62,7 @@ def __init__(self):
         self.data_function = None
         self.pixel_units = None
         self.plot_scalebar = True
+        self.scalebar_color = 'white'
```

Comment:
I agree, the default is not needed here, since it is set by default in `MPL_HyperExplorer`